### PR TITLE
Apply webkit-scrollbar to React modals

### LINKED ISSUE #2689: Scrollbars do not appear on Webkit Browsers on MacOSX
Specifically, to check out the create assessment modal (Might have others as well).

Reading: http://blog.0100.tv/2012/11/webkit-scrollbars-on-os-x/

### PR DESCRIPTION
**Pull request by fonglh**
Ensure that scrollbars show up in the create assessment modal,
create survey form modal and the create achievement modal
regardless of OS X's scrollbar settings.

The missing scrollbar has caused user confusion.

Fixes #2689.

### CODE REVIEW

**Comment by weiqingtoh** [comment]
These things should be applied globally since it affects all modals with of this nature. Can you shift it to somewhere where it is applied globally?

**Comment by fonglh** [comment]
There should be 3 relevant places which are using the `Dialog` modal from `material-ui`, and with a long enough modal that scrollbars might be needed.

1. `client/app/lib/components/FormDialogue.jsx`
2.  `client/app/bundles/course/achievement/pages/AchievementIndex/index.jsx`
3. `client/app/bundles/course/assessment/pages/AssessmentIndex/index.jsx`

File: client/app/lib/components/ModalForm.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+.modalForm {
+  $base-scrollbar-color: #000000;
+
+  &::-webkit-scrollbar {
+    width: 9px;
+  }
+
+  &::-webkit-scrollbar-track {
+    background: rgba($base-scrollbar-color, 0.1);
+    -webkit-border-radius: 5px;
```
[the remark this answers is not in the document]
Comment (fonglh) [reply client/app/lib/components/ModalForm.scss]: Won't fix, needed to make scrollbars appear.

File: client/app/lib/components/ModalForm.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    border-radius: 5px;
+  }
+
+  &::-webkit-scrollbar-thumb {
+    background: rgba($base-scrollbar-color, 0.2);
+    -webkit-border-radius: 5px;
```
[the remark this answers is not in the document]
Comment (fonglh) [reply client/app/lib/components/ModalForm.scss]: Won't fix, needed to make scrollbars appear.

File: client/app/lib/styles/ModalForm.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
+// scss-lint:disable ImportantRule
```
Comment (weiqingtoh) [inline client/app/lib/styles/ModalForm.scss]: I think there is no need to disable the important rule here. 

Would suggest to explain the nature of this file, and the source of where we get the css from.

**Pull request merged**